keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -397,6 +397,15 @@ class ApplicationService(
     )
   }
 
+  /**
+   * Translates the stored time window constraint format to numeric format for display
+   */
+  private fun translateAllowedTimesConstraint(constraint: TimeWindowConstraint) =
+    AllowedTimesConstraintMetadata(
+      AllowedTimesConstraintEvaluator.toNumericTimeWindows(constraint),
+      constraint.tz
+    )
```

Comment:
Maybe change the constructor of `AllowedTimesConstraintMetadata` to  just take the constraint as a parameter?

Reply:
Done.